steps/generate_analytical_dataset.py

```diff
@@ -424,5 +421,5 @@ def get_tables(db_name):
     main()
     end_time = time.perf_counter()
     total_time = round(end_time - start_time)
-    total_time = str(datetime.timedelta(total_time))
+    total_time = str(datetime.timedelta(seconds=total_time))
```

Comment:
Does this need to be `datetime.datetime.timedelta` like the `strptime`/`strftime` changes above?

Reply:
no this one doesn't. timedelta belongs to the main datetime class whereas .strptime belongs to the subclass .datetime 

not sure why they have this naming convention.